grakn-test/src/test/java/ai/grakn/test/graql/reasoner/ReasoningTests.java

```diff
@@ -46,7 +48,7 @@ public void testSet10() {
         QueryBuilder iqb = testSet10.graph().graql().infer(true);
```

Comment:
All I am asking is that we detail what "meanders of reasoning" are being tested in this case. 
You say it is testing what this PR is about then say in a javadoc or the test name "testing relation equivalence by doing x"

From the outsiders perspective I have no idea why this test should exist so when I break it I might be tempted to `@Ignore` and move on. I am simply trying to prevent that from happening by having clearly stated test cases.

If you changed this to something like:

```
/**
* Tests backward chaining in a scenario with a deep ontology.
* Tests lazy loading a large dataset
* Tests Relation equivalence etc . . . 
**/
public void testSet10(){}
```

That provides a lot more information as to where to start debugging if this test breaks.